frontend/src/components/teacher/student-management/classroom-summary/ClassroomCard.tsx

```diff
@@ -22,6 +22,7 @@ import ClassroomPopover from "./ClassroomPopover";
 
 interface ClassroomCardProps {
   id: string;
+  thin?: boolean;
```

Comment:
maybe rename to like `dashboardView`? smth a bit more encompassing of the two changes